test/cpp/qps/qps_worker.cc

```diff
@@ -223,7 +223,8 @@ QpsWorker::QpsWorker(int driver_port) {
   impl_.reset(new WorkerServiceImpl());
 
   char* server_address = NULL;
-  gpr_join_host_port(&server_address, "::", driver_port);
+  // TODO: we are not using "::" address to work around #4340.
+  gpr_join_host_port(&server_address, "localhost", driver_port);
```

Comment:
Question - doesn't this bind us just to the localhost IP address, in which case we lose our external IP addresses? That would be a problem for running this across machines. That was the reason for the original wildcard.